app/src/main/java/org/mozilla/fenix/perf/ApplicationInitTimeContainer.kt

```diff
@@ -0,0 +1,31 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.perf
+
+import android.os.SystemClock
+
+/**
+ *  A class to store the application initialization time.
+ *  Time is stores in elapsed real time nano seconds
+ */
+internal class ApplicationInitTimeContainer(
+    private val getElapsedRealtimeNanos: () -> Long = SystemClock::elapsedRealtimeNanos
+) {
+
+    var applicationInitNanos = -1L
```

Comment:
thought (non-blocking): in the removed code, there didn't seem to be any tests that test this functionality directly (that applicationInitNanos is set correctly) so the tests were correctly removed. However, this functionality could benefit from a new test. I don't feel so strongly about this that I want to block the PR but just a thought.

Reply:
I think I could add a test for it in `cleaning unused resource PR`